.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+# Description
+Please include a summary of the changes and the related issue. List any dependencies that are required for this change.
+
+# Linked Items:
+Please include the associated items of this Pull Request using the following syntax to link them for closure on completion of this Pull Request.
+`Closes #<Issue Number>`
+`Fixes #<Issue Number>`
+`Resolves #<Issue Number>`
+
+## Type of change
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Feature improvement
+- [ ] Requires a documentation update
+
+### Feature Additions
+Include pertinent changes as a result of adding the new feature.
+- None
+
+### Bug Fixes
+Include pertinent changes of fixing the bug.
+- None
+
+### Breaking Changes
+Include pertinent changes that qualify as breaking changes. Be sure to include information about the impact to end users and additional information for addressing the break if more than a version transition is necessary for mitigating. 
+- None
+
+### Feature Improvement Changes
+Include pertinent changes that are improvements to existing features.
+- None
+
+# Checklist:
+- [ ] Code follows the style guidelines of this project.
+- [ ] Code is commented (at minimum new code, but if legacy code is missing comments, try to update.)
+- [ ] Updated documentation accordingly.
+- [ ] My changes generate no new warnings.
+- [ ] Have updated new and existing unit tests accordingly.
+- [ ] Unit tests pass locally.
+- [ ] Pipeline completes and passes all stages.
+- [ ] Linked associated items to be closed.
```

Comment:
I think we can slim this down since we have automation to check for styling, unit tests passing locally (we already cover all major operating systems in the pipeline), pipeline passing will automatically block the PR, and linked items should be automatically closed out on this PR is closed.

```suggestion
- [ ] Code is commented (at minimum new code, but if legacy code is missing comments, try to update.)
- [ ] Updated documentation accordingly.
- [ ] Have updated new and existing unit tests accordingly.
```

Reply:
Requested removals make sense. The list was generated without having this project pipeline up as reference, so tried to cover the things that I've seen fall off on teams. 

Would argue to keep the linked items piece as a follow up for the section earlier in the template to help ensure it gets done since there aren't backup automation checks in for it yet. While the linked items will automatically be closed with completion of the PR, there isn't an assist for making sure items were linked to be closed.